inception-kb/src/main/java/de/tudarmstadt/ukp/inception/kb/SPARQLQueryStore.java

```diff
@@ -36,6 +39,7 @@
             , "}"
             , "LIMIT 10000");
     
+    // Add queries if needed for different KBs 
```

Comment:
What is this out commented code? Please delete it if not needed.

Reply:
Removed from this branch.